### PR TITLE
Feature/boas 460 adds storage blob data contributor role

### PR DESCRIPTION
**Pull request by mayya-bond-kc**

### CODE REVIEW

File: app/components/back-office-app-services/variables.tf [hunk 1]
```diff
@@ -167,3 +172,8 @@ variable "tags" {
   description = "The tags applied to all resources"
   type        = map(string)
 }
+
+variable "resource_group_id" {
```
Comment (lestermarch) [inline app/components/back-office-app-services/variables.tf:176]: This variable could be removed if we scope the RBAC permissions to the document store as opposed to the whole resource group.

File: app/stacks/uk-west/back-office/app-service.tf [hunk 2]
```diff
@@ -27,6 +28,7 @@ module "app_services" {
   monitoring_alerts_enabled                        = var.monitoring_alerts_enabled
   node_environment                                 = var.node_environment
   private_endpoint_enabled                         = var.private_endpoint_enabled
+  resource_group_id                                = azurerm_resource_group.back_office_stack.id
```
Comment (lestermarch) [inline app/stacks/uk-west/back-office/app-service.tf:31]: This variable could be removed if we scope the RBAC permissions to the document store as opposed to the whole resource group.

**Review by lestermarch — COMMENTED** [review]
Left some comments regarding scoping RBAC permissions. If you're ok with limiting permissions to the single document store then it might be neater. Otherwise disregard my comments and I'll approve.

File: app/components/back-office-app-services/iam.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+resource "azurerm_role_assignment" "applications_case_admin_officer_documents_access" {
+  scope                = var.resource_group_id
```
[the remark this answers is not in the document]
Comment (mayya-bond-kc) [reply app/components/back-office-app-services/iam.tf:2]: Aha! I was just copying what was done in Appeals. Let me change this now

**Review by lestermarch — APPROVED** [review]

**Pull request merged**